learning_resources/etl/sloan_test.py

```diff
@@ -84,3 +94,60 @@ def test_sloan_extract_programs(
     extracted = extract()
     assert extracted[0] == mock_sloan_courses_data
     assert extracted[1] == mock_sloan_runs_data
+
+
+def test_transform_run(
+    mock_sloan_courses_data,
+    mock_sloan_runs_data,
+):
+    run_data = mock_sloan_runs_data[0]
+    course_data = mock_sloan_courses_data[0]
+    assert transform_run(run_data, course_data) == {
+        "run_id": run_data["CO_Title"],
+        "start_date": parse_datetime(run_data["Start_Date"]),
+        "end_date": parse_datetime(run_data["End_Date"]),
+        "title": course_data["Title"],
+        "url": course_data["URL"],
+        "availability": RunAvailability.current.value,
+        "published": True,
+        "prices": [run_data["Price"]],
+    }
+
+
+def test_transform_course(mock_sloan_courses_data, mock_sloan_runs_data):
+    course_data = mock_sloan_courses_data[0]
+    runs_data = mock_sloan_runs_data[0:10]
+    course_runs_data = [
+        run for run in runs_data if run["Course_Id"] == course_data["Course_Id"]
+    ]
+
+    transformed = transform_course(course_data, runs_data)
+
+    assert transformed["readable_id"] == course_data["Course_Id"]
+    assert transformed["runs"] == [
+        transform_run(run, course_data) for run in course_runs_data
+    ]
+    assert transformed["learning_format"] == list(
+        {transform_format(run["Delivery"])[0] for run in course_runs_data}
+    )
+    assert transformed["image"] == parse_image(course_data)
+    assert sorted(transformed.keys()) == sorted(
+        [
+            "readable_id",
+            "title",
+            "offered_by",
+            "platform",
+            "etl_source",
+            "description",
+            "url",
+            "image",
+            "certification",
+            "certification_type",
+            "professional",
+            "published",
+            "learning_format",
+            "topics",
+            "course",
+            "runs",
+        ]
+    )

```

Comment:
Would be good to test for specific values for some of these, like:
```python
assert transformed["image"] == {
    "url": <expected url>,
    "alt": <expected alt string>,
    "description": <expected description>"
 }
 ```
 
 Could do so in separate tests instead like `test_parse_image`, `test_parse_topics` etc if that is easier.

Reply:
added tests for both.